langchain/prompts/prompt.py

```diff
@@ -51,6 +52,12 @@ def format(self, **kwargs: Any) -> str:
         """
         return DEFAULT_FORMATTER_MAPPING[self.template_format](self.template, **kwargs)
 
+    def _prompt_dict(self) -> Dict:
+        """Return a dictionary of the prompt."""
```

Comment:
this can probably go on base right? and then few-shot can subclass and call super